crates/nu-command/src/filters/where_.rs

```diff
@@ -19,7 +19,11 @@ impl Command for Where {
 
     fn signature(&self) -> nu_protocol::Signature {
         Signature::build("where")
-            .required("cond", SyntaxShape::RowCondition, "condition")
+            .required(
+                "block",
+                SyntaxShape::Block(Some(vec![SyntaxShape::Any])),
```

Comment:
Hmm... Change that syntax shape to `Block` will break the `$it` usage inside where

Reply:
Ah, that is an issue. I'll draft the PR for the moment and try to find a way to resolve this.